src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md

```diff
@@ -0,0 +1,249 @@
+
+<CodeGroup labels={["Before: The previous k6 browser API"]} lineNumbers={[true]}>
+
+```javascript
+import { chromium } from 'k6/experimental/browser';
+
+export default async function () {
+    const browser = chromium.launch({
+        headless: false,
+        timeout: '60s',
+    });
+    const page = browser.newPage();
+
+    try {
+        await page.goto('https://test.k6.io/');
+        page.screenshot({ path: 'screenshot.png' });
+    } finally {
+        page.close();
+        browser.close();
+    }
+}
+```
+
+</CodeGroup>
+
+<CodeGroup labels={["After: The new k6 browser API"]} lineNumbers={[true]}>
+
+```javascript
+import { browser } from 'k6/experimental/browser';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+          type: 'chromium',
+        },
+      },
+    },
+  },
+  thresholds: {
+    checks: ["rate==1.0"]
+  }
+}
+
+export default async function () {
+  const page = browser.newPage();
+
+  try {
+    await page.goto('https://test.k6.io/');
+    page.screenshot({ path: 'screenshot.png' });
+  } finally {
+    page.close();
+  }
+}
+```
+
+</CodeGroup>
+
+
+
+
+
+## Import Path
+
+Changes have been made to the way the [browser module](/javascript-api/k6-experimental/browser/) is imported. With the browser type (i.e., `chromium`) now set in [scenario options](#scenario-options), users should directly import the [browser](/javascript-api/k6-experimental/browser/#browser-module-api) object from the [browser module](/javascript-api/k6-experimental/browser/). In the past, users would rely on `chromium.launch()` for accessing the running browser. Now, a simple import of the [browser](/javascript-api/k6-experimental/browser/#browser-module-api) sufficient.
+
+<CodeGroup labels={["Before: Import Path"]} lineNumbers={[true]}>
+
+```javascript
+import { chromium } from 'k6/experimental/browser';
+```
+
+</CodeGroup>
+
+<CodeGroup labels={["After: Import Path"]} lineNumbers={[true]}>
+
+```javascript
+import { browser } from 'k6/experimental/browser';
+```
+
+</CodeGroup>
+
+
+
+
+
+## Browser options
+
+In the updated version, the need to manually start a browser and set its configuration has been removed, and this part can simply be omitted from test scripts.
+
+Users can still change some browser settings by using environment variables. For more information, refer to the [browser module options](/javascript-api/k6-experimental/browser/#browser-module-options) documentation.
+
+
+<CodeGroup labels={["Before"]} lineNumbers={[true]}>
+
+<!-- eslint-skip -->
+
+```javascript
+export default async function () {
+    const browser = chromium.launch({
+        headless: false,
+        timeout: '60s',
+    });
+}
+```
+
+</CodeGroup>
+
+<Blockquote mod="note" title="">
+
+The following browser options are no longer supported: `slowMo`, `devtools`, `env`, and `proxy`.
```

Comment:
Do we need a quick explanation here why they are no longer supported?

Reply:
👍 Makes sense. Done.